doc/en/multitest.rst

```diff
@@ -1026,18 +1029,23 @@ method has default values assigned to the parametrized arguments:
 Testcase name generation
 ++++++++++++++++++++++++
 
-When you use parametrization, Testplan will try to generate a legible name for
+When you use parametrization, Testplan will try to generate a function name for
 the test case methods generated. By default the name format is
 ``<original_method_name>__<arg_1_name>_<arg_1_value>__...__<arg_n_name>_<arg_n_value>``.
 If the generated method name is longer than 255 characters or does not have a
 valid Python attribute name format (alphanumeric with underscores), Testplan
 will fall back to simple format: ``<original_method_name>__<integer_suffix>``.
 
-In the example below, 2 new testcases will be generated, with the names being
-``'test_add_list__number_list_[1, 2, 3]__expected_6'`` and
-``'test_add_list__number_list_[6, 7, 8, 9]__expected_30'``. These are not valid
-Python attribute names, so Testplan will fall back to ``'test_add_list__0'`` and
-``'test_add_list__1'``.
+Each parametrized testcase also have a legible name used for display (usually called
```

Comment:
"When you use parametrization, Testplan will try to create a sensible name for each generated testcase. By default the naming convention is:
``original_testcase_name <param1=value1, param2=value2, ...>``

User can provide custom name generation functions to override this default:
...

Reply:
fixed